packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -475,24 +492,18 @@ async function beforeTests() {
     await client.closePrivacySettingsModal();
   });
 
-  return { keychain, compass };
+  return { compass };
```

Comment:
We might as well just `return compass;` now and then `afterTests(compass)` rather than `afterTests({ compass })`and have compass contain all the context going forward? What do you think?

Reply:
Yeah, makes sense, will change 👍

Reply:
Done in d5bf184bcf33275d1e44fada046594fd1124941f